UKHomeOffice/cerberus-service
# COP-4073 Bump formio-gds-template version

## Description
Bump formio-gds-template version, remove --watch as no longer supported/required

## To Test

- delete node modules
- delete package lock
- run npm i
- open Cerberus UI
- pages and forms should retain their styling

## Developer Checklist

\* Required
